### ISSUE TITLE
Rig's web server's SGV and graph are mangled

### ISSUE THREAD

**Issue by jaylagorio** [openaps/oref0#979]
Using G5, running the latest `dev` branch commits, the web server on the device shows the following display:

![image](https://user-images.githubusercontent.com/15005116/39096503-f0d00914-461e-11e8-8a51-507f8e641a93.png)

The loop is running properly and Nightscout looks as expected.

**Comment by jimrandomh** [comment]
I see similar issues on 0.7.0-dev (including "BG NaN").

**Comment by Tornado-Tim** [comment]
Seeing similar issues here on 0.6.1:
![openaps-webui](https://user-images.githubusercontent.com/2896311/39502551-f7e5a56c-4e14-11e8-9774-2e6ca3283359.png)
Subsequent refreshes, sometimes fixes the issue as it looks like below:
![openaps-webui-2](https://user-images.githubusercontent.com/2896311/39502563-0bbcf3f6-4e15-11e8-8991-5ec89a6edb5c.png)
But a refresh afterwards around 5 min later returns dud data again:
![openaps-webui-3](https://user-images.githubusercontent.com/2896311/39502586-1e09dc86-4e15-11e8-9119-c8fba3059292.png)

**Comment by tim2000s** [comment]
I got this on Dev
![screenshot_20180514-085510](https://user-images.githubusercontent.com/14833174/40013844-b279438a-57a6-11e8-95ad-b8a2ceb545da.png)

**Comment by jaylagorio** [comment]
I'm on dev, but I know I'm not seeing quite the detail in here that others are displaying. Above I see that there are entries for boluses, for example, but I've never seen those. Just updated the rig to the latest commit and things are still looking off:

![image](https://user-images.githubusercontent.com/15005116/40089324-2be72430-5879-11e8-8c67-12b5fcad7302.png)

Screenshot is Edge, but Firefox and Chrome look the same.

**Comment by Ashley112885** [comment]
I'm getting the NaN issue on my phone (Safari and Chrome), but not on my PC (Chrome). I'm using 0.7.0-dev with Logger.
![openaps](https://user-images.githubusercontent.com/38409940/40854939-fe6438b0-65a0-11e8-81a2-1422f982fbe2.jpg)
![openaps2](https://user-images.githubusercontent.com/38409940/40855086-71ac09f6-65a1-11e8-85ab-411094434153.JPG)

**Comment by danamlewis** [comment]
I don't think this was tested prominently with the latest release (master is now 0.7.0); just noting that here in case someone wants to use that as a motivation to tackle reviewing & improving the offline web server since it's not widely used.
